build.sh

```diff
@@ -260,6 +284,19 @@ case "$COMMAND" in
         ;;
 
     ######################################
+    # Swift versioning
+    ######################################
+    "set-swift-version")
+        version="$2"
+        if [[ -z "$version" ]]; then
+            version="$(xcrun swift --version 2>/dev/null | sed -ne 's/^Apple Swift version \([^\b ]*\).*/\1/p')"
+        fi
+        rm -f RealmSwift && ln -s "RealmSwift-swift$version" RealmSwift && (git update-index --assume-unchanged RealmSwift || true)
```

Comment:
Also needs a comment that briefly says what `git update-index` is using since I'm guessing it's not a command that most are familiar with.